packages/astro/src/core/build/common.ts

```diff
@@ -1,8 +1,10 @@
 import npath from 'path';
+import { fileURLToPath, pathToFileURL } from 'url';
 import type { AstroConfig, RouteType } from '../../@types/astro';
 import { appendForwardSlash } from '../../core/path.js';
 
 const STATUS_CODE_PAGES = new Set(['/404', '/500']);
+const FALLBACK_OUT_DIR_NAME = './.astro-temp/';
```

Comment:
I'd be fine with `.astro`, personally? Not a huge deal but it's a bit more clean.

```suggestion
const FALLBACK_OUT_DIR_NAME = './.astro/';
```

Reply:
I'm fine with this too. I thought that in the case where the build fails, and `.astro-temp` is not removed (for debugging), it sends a clearer message that it's a temporary folder that can be removed.

Reply:
That's fine with me! I'd probably add the directory to the `.gitignore` files in our examples just to be safe?

Reply:
@natemoo-re I updated to use `.astro`. I think a `.gitignore` change shouldn't be needed at the meantime since this special directory is only created if someone specifies a `outDir` outside of the project. Perhaps if there's a request for it you can add that too.